### ISSUE TITLE
Performance degradation from 4.3.0 -> HEAD

### ISSUE THREAD

**Issue by dgoeries** [mosquito/aiormq#156]
Hi, I am experiencing a significant performance decrease when trying aiormq with a simple sender and receiver.

On my test machine I can consume with version 4.3.0 still 6.5kHz of messages, with this little script. I have the sender independent on _pika_ to check. 

What could be the reason?

```python
import asyncio
import sys
from time import perf_counter

import aiormq


class MessageCounter:
    def __init__(self):
        self.counter = 0
        self.before = perf_counter()

    async def on_message(self, message):
        self.counter += 1
        if self.counter % 1000 == 0:
            print(f" [x] Received {self.counter} messages.")
            print(f" [x] {1000 / (perf_counter() - self.before)} Hz.")
            self.before = perf_counter()


async def main():
    counter = MessageCounter()
    connection = await aiormq.connect("amqp://")
    channel = await connection.channel()
    declare_ok = await channel.queue_declare("hello")
    channel.queue_purge(declare_ok.queue) # Working?
    consume_ok = await channel.basic_consume(
        declare_ok.queue, counter.on_message, no_ack=True)
    print(" [*] Waiting for messages. To exit press CTRL+C.")
    await asyncio.Future()


if __name__ == "__main__":
    try:
        asyncio.run(main())
    except KeyboardInterrupt:
        sys.exit(0)
```

```python
import sys
from time import sleep

import pika


def main():
    params = pika.URLParameters("amqp://")
    connection = pika.BlockingConnection(params)
    channel = connection.channel()
    channel.basic_qos(prefetch_count=1)
    channel.queue_declare(queue="hello")

    print("Start to send messages ...")
    while True:
        channel.basic_publish(
            exchange="", routing_key="hello",
            body=b"helloworld",
            mandatory=True)
        sleep(0.0000001)


if __name__ == "__main__":
    try:
        main()
    except KeyboardInterrupt:
        sys.exit(0)
```

**Comment by dgoeries** [comment]
Thanks! To add more information, with latest _aiormq_ the consumption goes down to 900Hz

**Comment by mosquito** [comment]
bottleneck [here](https://github.com/mosquito/aiormq/blob/master/aiormq/tools.py#L89), but right now I really have no idea how to fix this easily.

**Comment by mosquito** [comment]
Seems to have [come up with](https://github.com/mosquito/aiormq/pull/157)

**Comment by mosquito** [comment]
released as [6.5.0](https://pypi.org/project/aiormq/6.5.0/)
